modules/sdk/src/main/scala/org/tessellation/sdk/config/FixedAddresses.scala

```diff
@@ -0,0 +1,17 @@
+package org.tessellation.sdk.config
+
+import org.tessellation.schema.address.Address
+
+import eu.timepit.refined.auto._
+
+object FixedAddresses {
```

Comment:
This is fine to separate in a different file, but it should still carry the named LockedAddresses I think, and stardustSecondary should be left in the other configuration location since that is remaining untouched.